src/main/webapp/style.css

```diff
@@ -107,6 +107,18 @@ html {
   width: 50%;
 }
 
+/* Transcript */
```

Comment:
Yes, I was waiting for #133 to be merged into master to make that change since the `lecture-view.css` file doesn't exist in master yet. I added file and moved the code over for now. 

Done! 

Edit: It was just added so I resolve the merge conflict.